src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -106,16 +143,42 @@
   </ItemGroup>
   <!-- WINDOWS: CoreCLR -->
   <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(CoreClrOrCorRt)' == 'true'">
+    <Compile Include="System\Environment.CoreCLR.cs" />
```

Comment:
In other words, we want this implementation to be usable for CoreRT console apps - replacing what's there today (https://github.com/dotnet/corert/blob/master/src/System.Private.CoreLib/src/System/Environment.Win32.cs).